bc_obps/registration/migrations/0001_initial.py

```diff
@@ -0,0 +1,308 @@
+                        blank=True, db_comment='', max_length=128, region=None
+                    ),
+                ),
+                ('is_operational_representative', models.BooleanField(db_comment='')),
+                ('verified_at', models.DateTimeField(blank=True, db_comment='', null=True)),
+            ],
+            options={
+                'db_table_comment': "Contacts (people who don't use the app, e.g. authorized signing officers)",
+            },
+        ),
+        migrations.CreateModel(
+            name='Document',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('file', models.FileField(db_comment='', upload_to='documents')),
+                ('document_type', models.CharField(db_comment='Type of document, e.g., boundary map', max_length=1000)),
+                ('description', models.CharField(db_comment='', max_length=1000)),
+            ],
+            options={
+                'db_table_comment': 'Documents',
+            },
+        ),
+        migrations.CreateModel(
+            name='NaicsCode',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('naics_code', models.CharField(db_comment='', max_length=1000)),
+                ('ciip_sector', models.CharField(db_comment='', max_length=1000)),
+                ('naics_description', models.CharField(db_comment='', max_length=1000)),
+            ],
+            options={
+                'db_table_comment': 'Naics codes',
+            },
+        ),
+        migrations.CreateModel(
+            name='Operator',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('legal_name', models.CharField(db_comment='', max_length=1000)),
+                ('trade_name', models.CharField(db_comment='', max_length=1000)),
+                ('cra_business_number', models.CharField(db_comment='', max_length=1000)),
+                ('bc_corporate_registry_number', models.CharField(db_comment='', max_length=1000)),
+                ('business_structure', models.CharField(db_comment='', max_length=1000)),
+                ('mailing_address', models.CharField(db_comment='', max_length=1000)),
+                ('bceid', models.CharField(db_comment='', max_length=1000)),
+                (
+                    'relationship_with_parent_operator',
+                    models.CharField(blank=True, db_comment='', max_length=1000, null=True),
+                ),
+                ('date_aso_became_responsible_for_operator', models.DateTimeField(db_comment='')),
+                (
+                    'compliance_obligee',
+                    models.ForeignKey(
+                        db_comment='',
+                        on_delete=django.db.models.deletion.CASCADE,
```

Comment:
We may decide want some `on delete cascade` effects, but for now lets default to `do nothing` until we're sure we want cascading delete actions.